helpers/helpers_emba_print.sh

```diff
@@ -739,7 +739,8 @@ print_notification(){
   local CURRENT=""
   CURRENT=$(<"$NOTIFICATION_LOCATION")
 
-  inotifywait -m -e modify "$NOTIFICATION_LOCATION" --format "%e" | while read -r EVENT; do
+  disable_strict_mode "$STRICT_MODE" 0
```

Comment:
I am missing a `enable_strict_mode "$STRICT_MODE" 0` here

Reply:
As this process get killed on EMBA exit it is not needed

Reply:
Oh, that's true. I didn't think of that 😄